apiary-receivers-parent/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Overview
+
+`apiary-receivers-parent` provides common libraries for receiving messages from Apiary messaging infrastructure.  
```

Comment:
`apiary-receivers-parent` -> `apiary-receivers` (here and in folder name and all references etc.)

Reply:
Done